orbstation/clothing/misc.dm

```diff
@@ -0,0 +1,12 @@
+/obj/item/clothing/glasses/boggle
+	name = "boggle goggles"
+	desc = "A bandaid solution to the universe banning wearing eyes on eyes."
```

Comment:
This is not too important I would maybe prefer a less metatextual description :p
Something like "Now you can put eyes on your eyes for more eye per eye."